packages/utils/src/extendedProfileTypes.ts

```diff
@@ -118,9 +118,9 @@ export type CeramicEPObjects = EPObjects & {
 
 export type HasuraStringProps = HasuraBPStrings & HasuraEPStrings;
 export type HasuraImageSourcedProps = {
-  -readonly [key in keyof typeof BasicProfileImages]?: ImageSources;
+  -readonly [key in keyof typeof BasicProfileImages]?: ImageSources | null;
```

Comment:
In most of the rest of this code I used `Maybe<>` to represent that a field is nullable.
```suggestion
  -readonly [key in keyof typeof BasicProfileImages]?: Maybe<ImageSources>;
```